x-pack/plugins/security_solution/public/app/solution_navigation/links/sections/icons/notes.tsx

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import type { SVGProps } from 'react';
+import React from 'react';
+
+export const IconNotes: React.FC<SVGProps<SVGSVGElement>> = ({ ...props }) => (
+  <svg
```

Comment:
If you do happen to use this SVG markup, I'd suggest you compress its size using https://svgomg.net/. 

Here's the compressed version that saves up to 89 bytes but renders the same
```json5
<svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 32 32" {...props}>
  <path fill="#017D73" d="M15 13H6v2h9v-2Zm7 5H6v2h16v-2ZM6 23h16v2H6v-2Z"/>
  <path fill="#343741" d="M19.41 0H3a3 3 0 0 0-3 3v26a3 3 0 0 0 3 3h22a3 3 0 0 0 3-3V8.59L19.41 0ZM20 3.41 24.59 8H20V3.41ZM25 30H3a1 1 0 0 1-1-1V3a1 1 0 0 1 1-1h15v8h8v19a1 1 0 0 1-1 1Z"/>
</svg>
```

Settings I used with your markup to compress (precision 3, plus swithces)
![Screenshot 2024-10-03 at 09 37 19](https://github.com/user-attachments/assets/a1f2125b-93bf-403d-b2de-5c236aaab5c6)